hledger-lib/Hledger/Data/Journal.hs

```diff
@@ -532,33 +534,14 @@ journalReverse j =
     ,jmarketprices     = reverse $ jmarketprices j
     }
 
+-- | Set clock time in journal
 journalSetTime :: ClockTime -> Journal -> Journal
 journalSetTime t j = j{ jlastreadtime = t }
 
+-- | See filepath in journal
```

Comment:
"Add a file (and its content) to the journal's list of parsed files. (Does not parse the file or change the journal otherwise.)" ? Rename to journalAddFile or similar ? Move next to journalFilePaths ?

Reply:
Actually, there's already a redundant `journalAddFile` in `Read.Common`. The only difference is that it adds to the end. We don't need both, since we can just use the one on Read.Common and put it prior to the reversal.

Reply:
Addressed in 0d2f8c4 below.